docs/new_instrument.rst

```diff
@@ -179,15 +214,52 @@ Using the CDAWeb template the equivalent action is
 
 .. code:: python
 
-   download = functools.partial(nasa_cdaweb_methods.download, 
+   download = functools.partial(nasa_cdaweb_methods.download,
                                 supported_tags)
 
 where supported_tags is defined as dictated by the download function. See the routines for cnofs_vefi and cnofs_ivm for practical uses of the NASA CDAWeb support code.
 |br|
 
 
 .. automodule:: pysat.instruments.nasa_cdaweb_methods
-   :members: 
+   :members:
```

Comment:
There is a formatting issue that starts here. The stuff below is rendered as is, not as formatted text.

Reply:
fixed